d.ts/flv.d.ts

```diff
@@ -141,39 +244,73 @@ declare namespace FlvJs {
     }
 
     interface Player {
-        constructor: PlayerConstructor;
         destroy(): void;
-        on(event: string, listener: () => void): void;
-        off(event: string, listener: () => void): void;
+        on(event: string, listener: (...args: any[]) => void): void;
+        off(event: string, listener: (...args: any[]) => void): void;
         attachMediaElement(mediaElement: HTMLMediaElement): void;
         detachMediaElement(): void;
         load(): void;
         unload(): void;
-        play(): Promise<void>;
+        play(): Promise<void> | void;
         pause(): void;
         type: string;
         buffered: TimeRanges;
         duration: number;
         volume: number;
         muted: boolean;
         currentTime: number;
-        mediaInfo: AnyObject;
-        statisticsInfo: AnyObject;
+        mediaInfo: NativePlayerMediaInfo | FlvPlayerMediaInfo;
+        statisticsInfo: NativePlayerStatisticsInfo | FlvPlayerStatisticsInfo;
+    }
+
+    interface NativePlayerStatisticsInfo {
+        playerType: 'NativePlayer';
+        url: string;
+        decodedFrames?: number;
+        droppedFrames?: number;
+    }
+
+    interface FlvPlayerStatisticsInfo {
+        playerType: 'FlvPlayer';
+        decodedFrames?: number;
+        droppedFrames?: number;
+    }
```

Comment:
There're more fields in FlvPlayer's StatisticsInfo
https://github.com/Bilibili/flv.js/blob/master/src/core/transmuxing-controller.js#L418